packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -349,6 +349,46 @@ test("The compiler can handle simple helpers", function() {
   compilesTo('<div>{{testing title}}</div>', '<div>hello</div>', { title: 'hello' });
 });
 
+test("GH#13999 The compiler can handle simple helpers with inline null parameter", function() {
+  env.registerHelper('say-hello', function() {
+    return 'hello';
+  });
+
+  compilesTo('<div>{{say-hello null}}</div>', '<div>hello</div>');
+});
+
+test("GH#13999 The compiler can handle simple helpers with inline undefined parameter", function() {
+  env.registerHelper('say-hello', function() {
+    return 'hello';
+  });
+
+  compilesTo('<div>{{say-hello undefined}}</div>', '<div>hello</div>');
```

Comment:
I suspect that this is not something we actually support. The wire format is converted into json, which doesn't support `undefined` as a value. `undefined` feels very JavaScript-centric, whereas Handlebars try to stay a bit more host-language-neutural in general (there are implementations in other languages, and having `undefined` vs `null` in those environments would feel quite out-of-place).